stable/elasticsearch-exporter/values.yaml

```diff
@@ -44,6 +44,9 @@ service:
   annotations: {}
   labels: {}
 
+## Extra environment variables that will be passed into the exporter pod
+# env:
```

Comment:
Could you give a small example? It can be done in a subsequent PR if you want to get this merged now.

Reply:
Good idea, I've added an example in https://github.com/helm/charts/pull/13981/commits/6bfacbfaf9f05b3593fce9fd4c8ba8ce7eddc36a